content/300-guides/100-performance-and-optimization/095-tracing.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: 'Tracing'
```

Comment:
Why not if they're two different URLs?

Reply:
Because Google still prefers to have clearly named pages as it can keep them apart. If you imagine these two pages to appear in search results, the user could not make a decision between the "Guide" and the "Concept description" page based on the title. I think our guides are usually named more elaborate "How to use Tracing in Prisma ORM" maybe or something like that.

Reply:
Yes, we definitely need unique titles (names of pages), not only for google but also so that we can refer to each page, if needed, by  a unique name. Also, as Jan points out, we do typically use longer, expanded titles; the reason  is for SEO... having a longer title allows us to create more possible hits for SEO. A title like "Using tracing for optimization" (`/using-tracing-for-optimization` in URL) for example, crams the word optimization in there. ;)

Reply:
Thanks for the info! That makes sense. For now I kept the main tracing doc as Tracing, but renamed the other one to "Performance Tracing". I wanted it to fit on one line in the sidebar.